Project-SAP/project-sap-backend
# SAP-16 Add database configuration and initialization

**Description**
Changes created database environment and established database connection with our web application. Database can be accessed via MongoDB Atlas web application for user interface. Everyone currently in Project-SAP has access to the database as a project owner, meaning full access to the database. 

Link to JIRA ticket: https://peakashu.atlassian.net/browse/SAP-16?atlOrigin=eyJpIjoiNmNhNjc4NWVjOTkzNGNlYWI0Mzk0NGM5ZTJmNGRiZjIiLCJwIjoiaiJ9

**How to Test**

1. Log in to Atlas (https://account.mongodb.com/account/login?nds=true&_ga=2.197338671.1429200999.1665445311-1652157088.1665445311) using your emails provided. 
2. Click on 'Browse Collections' in the 'Database' tab and you should see temporary example data within our database.
3. Update `.env` to be within config directory/folder and copy new environment variables in #important Discord channel.
4. Run `npm start` and no error should pop out. This shows that authentication succeeded and database is connected with the application.
